service-python/assessclaimdc7101/src/lib/continuous_medication.py

```diff
@@ -25,9 +13,9 @@ def continuous_medication_required(request_body):
 
   veterans_medication = request_body["evidence"]["medications"]
   for medication in veterans_medication:
-    medication_display = medication["description"]
-    for keyword in [x.lower() for x in hypertension_medications]:
```

Comment:
It is a bit odd that we are lower casing a constant list. Can't the list be just lower cases?

Reply:
This line was removed, but yes I agree